### ISSUE TITLE
Remove unused CSS

### ISSUE THREAD

**Issue by unclesam2024** [web-infrastructure/doc_architecture#28]
Remove CSS which isn't needed during the initial load of the page.

Specifically, CSS rules which don't apply to above-the-fold content during the initial load of the page should be inlined, i.e. minified and written within the `<style>` block of the `<head>` of the page, e.g.

<style>
body {background-color: #ADD8E6;color: #444; ...}
</style>

Non-critical CSS should be deferred using a [preload link](https://developer.mozilla.org/en-US/docs/Web/HTML/Link_types/preload) with the following pattern:

<link rel="preload" href="styles.css" as="style" onload="this.onload=null;this.rel='stylesheet'">
<noscript><link rel="stylesheet" href="styles.css"></noscript>
Learn more about [deferring non-critical CSS](https://web.dev/defer-non-critical-css/)
